test/test_datapipe.py

```diff
@@ -377,6 +377,72 @@ def fn(item, dtype=torch.float, *, sum=False):
         for x, y in zip(map_dp_nl, input_dp_nl):
             self.assertEqual(x, torch.tensor(y, dtype=torch.float))
 
+    def test_map_datapipe_nested_level(self):
+
+        input_dp = IDP([list(range(10)) for _ in range(3)])
+
+        def fn(item, dtype=torch.float, *, sum=False):
```

Comment:
sum kwarg doesn't ever seem to be exercised?

Reply:
It was used in a previous test but no longer needed here. I will remove it.